assets/src/application/ui/layout/entityList/filterBar/style.scss

```diff
@@ -158,37 +165,10 @@ $filter-btn: '.ee-filter-bar__btn';
 		}
 	}
 
-	/* Media Queries for specific screen sizes
- 	* -------------------------------------------------------------------------*/
-
-	@media only screen and (max-width: 1920px) {
-	}
-
-	@media only screen and (max-width: 1600px) {
-	}
-
-	@media only screen and (max-width: 1366px) {
-	}
-
 	@media only screen and (max-width: 1280px) {
 		margin: 0 0 var(--ee-margin-default);
 	}
 
-	@media only screen and (max-width: 1024px) {
-	}
-
-	/* WordPress Admin Media Query Breakpoint: Auto-folding of the admin menu */
-	@media only screen and (max-width: 960px) {
-	}
-
-	/* WordPress Admin Media Query Breakpoint: one column on the post write/edit screen */
-	@media only screen and (max-width: 850px) {
-	}
-
-	/* WordPress Admin Media Query Breakpoint: Tablet */
-	@media screen and (max-width: 782px) {
-	}
-
```

Comment:
hope this is ok, will prevent a lot of scrolling around
and is more maintanable